tests/lean/415.lean.expected.out

```diff
@@ -1,4 +1,4 @@
 Set.insert (Set.insert (Set.insert Set.empty 1) 2) 3 : Set Nat
-fun (x y : Nat) => g { x := x, y := y } : Nat → Nat → Nat
-fun (x y : Nat) => Set.insert (Set.insert Set.empty x) y : Nat → Nat → Set Nat
-fun (x y : Nat) => { x := x, y := y } : Nat → Nat → Point
+fun x y => g { x := x, y := y : Point } : Nat → Nat → Nat
```

Comment:
Structure instance types seem to be the most common additions in the test diffs. I see this is from the new `getPPStructureInstanceType` default `getPPAll o || getPPAnalyze o`, does this one still make sense with the latter now being defaulted to `true`? I would expect the instance type to be shown only when needed.

Reply:
Same with `pp.proofs`

Reply:
Good catch. f25b15189985f20011fe9662aa8dbcf2323059e3 removes most structure instance type annotations. I am not sure what you mean re: `pp.proofs`. By default, `pp.analyze` prints proofs like any other terms (i.e. `pp.proofs = true`), and so `pp.proofs.withType` is never considered.

Reply:
Nice.

> By default, `pp.analyze` prints proofs like any other terms (i.e. `pp.proofs = true`)

Which is a different default from before, correct? Printing proofs is necessary for round-tripping, of course, but I still don't think we want to show them in the UI by default.

Reply:
https://github.com/dselsam/lean4/commit/620a74e529af4e6e2a92158bb01eb2a3ab7c9fa9 sets the default `pp.proofs = false` and https://github.com/dselsam/lean4/commit/5974586e1aae4dd8ee65b7d864a022f146a53e12 makes sure the type of the proof will be analyzed if it is going to be delaborated.